phoenix-pherf/config/datamodel/timeout_test_schema.sql

```diff
@@ -0,0 +1,5 @@
+ CREATE TABLE IF NOT EXISTS PHERF.USER_DEFINED_TEST (
```

Comment:
This is test-only, so it should be in the test directory, for ex inside `https://github.com/apache/phoenix/tree/master/phoenix-pherf/src/test/resources/datamodel`